### PR TITLE
feat: added my blog post: godsakani

### PR DESCRIPTION
**Pull request by godsakani**
# Changes
- Added my multiple blog post

### CODE REVIEW

**Review by spykelionel — COMMENTED** [review]
Just to be clear. Is this a tech related blog post?

**Comment by godsakani** [comment]
ok

**Comment by godsakani** [comment]
> Just to be clear. Is this a tech related blog post?

This was not specify

File: blogs/godsakani.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      "Description": "Blogs tells us 14 ways to create privacy in our homes",
+      "URL": "https://skintdad.co.uk/cheap-ways-to-block-neighbors-view/",
+      "Author": " NAOMI WILLIS"
+    },
+      {
+      "Title": "How to Promote Local Small Businesses – 5 Proven Effective Methods!",
+      "Description": "This blog tells us how to better manage our savings",
+      "Link": "https://article.advertiseera.com/how-to-promote-local-small-businesses-5-proven-effective-methods/",
+      "Author": "STAFFUSER"
+    }
```
Comment (spykelionel) [inline blogs/godsakani.json:14]: Please Is this tech related?
Just to be clear

**Comment by spykelionel** [comment]
[Check this r readme](https://github.com/Developer-Student-Clubs-UBa/blog-posts)

**Review by spykelionel — APPROVED** [review]

File: blogs/godsakani.json (hunk 1 above)
Comment (godsakani) [reply blogs/godsakani.json]: ok sir

**Review by godsakani — COMMENTED** [review]
ok sir

**Review by yokwejuste — APPROVED** [review]
